package.json

```diff
@@ -33,10 +33,10 @@
     "@babel/plugin-proposal-class-properties": "^7.2.3",
     "@babel/preset-env": "^7.2.3",
     "@babel/preset-react": "^7.0.0",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^10.0.6",
-    "@testing-library/react-hooks": "^3.2.1",
-    "@testing-library/user-event": "^12.0.7",
+    "@testing-library/jest-dom": "^5.11.1",
```

Comment:
the major version bump is a bit scary but if we're green then it's good. do we have any deprecation warnings? or extra output when running tests? it might be a good opportunity to clean it all up